daofund-score/src/main/java/com/iconbet/score/daofund/DaoFund.java

```diff
@@ -0,0 +1,303 @@
+package com.iconbet.score.daofund;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+import score.Address;
+import score.ArrayDB;
+import score.Context;
+import score.DictDB;
+import score.VarDB;
+import score.annotation.EventLog;
+import score.annotation.External;
+import score.annotation.Payable;
+
+public class DaoFund {
+	
+	public String TAG = "ICONbet DAOfund";
+	
+    private static final String ADMINS = "admins";
+    private static final String WITHDRAW_COUNT = "withdraw_count";
+    private static final String WITHDRAW_RECORD = "withdraw_record";
+    private static final BigInteger X_6 = new BigInteger("1000000"); // 10 ** 6
+    private static final BigInteger BATCH_SIZE = BigInteger.valueOf(100L);
+    
+
+	private final ArrayDB<Address> admins = Context.newArrayDB(ADMINS, Address.class);
+	private final VarDB<BigInteger> withdraw_count = Context.newVarDB(WITHDRAW_COUNT, BigInteger.class);
+	private final DictDB<String,LinkedList> withdraw_record = Context.newDictDB(WITHDRAW_RECORD, LinkedList.class);
+
+	
+	/***
+	  def on_update(self) -> None:
+	        super().on_update()
+	        self.admins.put(self.owner)
+	        self.withdraw_count.set(0)
+	 * @param <T>
+	  ***/
+	
+	
+	private <T> boolean remove_array_item(T value, ArrayDB<T> arraydb ) {
+		
+		return Boolean.FALSE;
+		
+	}
+		
+	
+	/***
+	
+	def remove_array_item(array_db, target) -> bool:
+	    _out = array_db[-1]
+	    if _out == target:
+	        array_db.pop()
+	        return True
+	    for index in range(len(array_db) - 1):
+	        if array_db[index] == target:
+	            array_db[index] = _out
```

Comment:
very confusing py logic, we can to test it python cli and convert it to java, it is not implemented yet (line 44)
anyways we need to confirm from iconbet devs if the logic is correct

Reply:
Added missing method.